### DISCUSSION TITLE
What do I submit to get the Student Developer Pack? My school email domain doesn't work and my Student ID has no date.

### DISCUSSION THREAD

**Discussion by atuschek0** [community/community#28413] [GitHub Education]
Hello all,

I am currently trying to apply for the GitHub Student Developer pack. I am a student at a public community college in California that plans to transfer to the University of California - Irvine in Fall 2023 but the email box on the application said that my school email domain cannot be used. I was going to then submit my student ID although it has no date.

Under the "Or another form of proof" section it says to submit any documentation with a date to show student enrollment, I have already registered for my fall classes, should I just submit that? Or is there a better document i should send? I just want to submit a document that will be the fastest route to getting the developer pack, mainly because I want to apply to be a Github Campus Expert at my school before the deadline in September.

**Comment by mlarasusan** [comment]
Hello :] You can find some information here in this [page](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) about documents you can use ✅
